ext/sqlite3/sqlite3.stub.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+class SQLite3
+{
+    function open(string $filename, int $flags = SQLITE3_OPEN_READWRITE | SQLITE3_OPEN_CREATE, ?string $encryption_key = null): void;
+
+    function close(): void;
+
+    function version(): void;
+
+    function lastInsertRowID(): void;
+
+    function lastErrorCode(): void;
+
+    function lastExtendedErrorCode(): void;
+
+    function lastErrorMsg(): void;
+
+    function changes(): void;
+
+    function busytimeout(int $ms): bool;
+
+#ifndef SQLITE_OMIT_LOAD_EXTENSION
```

Comment:
Anybody know what I've done wrong with these that's causing them to not be included in the generated header?

Reply:
This was a bug in the generator, should be fixed with https://github.com/php/php-src/commit/ae721c488dc31c59a5a05b4220564ee8dbf8fbb0.